Cipher.cs

```diff
@@ -33,9 +33,11 @@ public string Encrypt(string message)
         string newMessage = string.Empty;
         foreach (char c in message)
         {
+            c = (char)('A' + this.shift);
+            Console.WriteLine($"'A' + this.shift = '{c}'");
```

Comment:
You're super close! You will want to use `newMessage += (char)(c + this.shift);`

Think about what that code does before putting it in.